bitcoin-magazine/bitcoin-explained/bitcoin-core-22.0-explained.md

```diff
@@ -0,0 +1,867 @@
+Your node can connect outbound but other nodes can't connect to you because they can't see you.
+And the way you solve that is by opening a port on the router, forwarding that port to your node.
+Which you can do manually, like you go to 192.168.0.2 and then you log in and you say port forwarding and you say send port A333 to this device that runs my node.
+But nobody wants to do that.
+So there are libraries that are often used by games and by programs like BitTorrent and all these things that will do it for you automatically.
+That will just, you know, from your computer ping your router and say please route this port to me.
+And that was done using UPnP, but apparently that's very insecure and very bad, so it was turned off by default in Bitcoin core, so nobody was using it.
+And now there's another library called NAT-PMP that apparently is a bit safer, a bit easier, and that's now added.
+So if you open Bitcin core in the settings in the GUI there's a button that says NAT-PMP.
+I think you can click on it and it'll try and then it'll try to open the port on the router to listen.
+I don't know if it gives any useful feedback, if it doesn't work.
+Great.
+I haven't tried it.
+I don't think it works on my router.
+
+Aaron: 00:33:11
+
+And then other than that, what we haven't mentioned, of course, every Bitcoin Core upgrade has a ton of smaller improvements.
+Yeah.
+Bug fixes, these kinds of things.
+But I think we agree that these were, what we've mentioned now, were the six most notable changes in this new Bitcoin core release.
+
+Sjors: 00:33:30
+
+Yeah, there's one meta change I think we forgot to mention, which is not inside the build itself, but is that the deterministic builds for this release are being made using Geeks for the first time.
+
+Aaron: 00:33:43
+
+Right, yeah, I do think we've mentioned that, but maybe you can explain real quick what that is.
+
+Sjors: 00:33:48
+
+Well, I think we talked about, I think we talked about reproducibility in an earlier episode, why open source matters, episode 21.
+We talked about, it's important for Bitcoin Core to be open source but also that the thing you're downloading from bitcoincore.org is actually related to that source that's out there.
+Because we could just say hey look at this beautiful source code and then give you some binary that's just ransomware.
+We spent the whole episode 21 explaining how to prevent that and GeeksGUIX is a new project, relatively new, that makes that process of verifiability better.
+We're now using that for the release.
+
+Aaron: 00:34:34
+
+Great.
+
+Sjors: 00:34:35
+
+Yeah I think that's that's basically it.
+
+Aaron: 00:34:38
+
+Yeah so the main changes were hardware wallet hardware wallet supported in the GUI, I2P support, first release to have taproot which includes receiving very basic taproot transactions.
+Testmempool accept was updated as an incremental step towards package relay.
```

Comment:
Here, "Testmempool accept" refers to the `testmempoolaccept` [RPC](https://bitcoincore.org/en/doc/22.0.0/rpc/rawtransactions/testmempoolaccept/). Ensure code-related technical terms or math equations are enclosed in backticks (`) to enhance clarity.